event_webhook_types.go

```diff
@@ -868,7 +868,22 @@ type SnippetCommentEvent struct {
 		Description  string `json:"description"`
 		URL          string `json:"url"`
 	} `json:"object_attributes"`
-	Snippet *Snippet `json:"snippet"`
+	Snippet *struct {
+		ID                 int        `json:"id"`
+		Title              string     `json:"title"`
+		Content            string     `json:"content"`
+		AuthorID           int        `json:"author_id"`
+		ProjectID          int        `json:"project_id"`
+		CreatedAt          string     `json:"created_at"`
+		UpdatedAt          string     `json:"updated_at"`
+		Filename           string     `json:"file_name"`
+		ExpiresAt          string     `json:"expires_at"`
```

Comment:
This field is documented but doesn't exist in the payload… 🤷‍♀️ not sure what to do. It looks like a docs error since I don't see any way to define an expiration date in the UI

Maybe: (?)

```suggestion
```